internal/app/encryption.go

```diff
@@ -29,24 +32,13 @@ func FindIndex(vs []string, t string) int {
 
 // Password ..
 func Password() string {
-	mathrand.Seed(time.Now().UnixNano())
-	chars := []rune("ABCDEFGHIJKLMNOPQRSTUVWXYZ" +
-		"abcdefghijklmnopqrstuvwxyz" +
-		"0123456789" +
-		"=+%*/()[]{}/!@#$?|")
+
 	length := viper.GetInt("server.generatedPasswordLength")
-	if length < 4 {
-		length = 4
-	}
-	var b strings.Builder
-	for i := 0; i < length; i++ {
-		randomNumber, err := rand.Int(rand.Reader, big.NewInt(int64(len(chars))))
-		if err != nil { // Fallback if something bad happens
-			randomNumber = big.NewInt(int64(mathrand.Intn(len(chars))))
-		}
-		b.WriteRune(chars[randomNumber.Int64()])
+	res, err := password.Generate(length, 10, 10, false, false)
+	if err != nil {
+		log.Fatalf("%s, error: %s", UnableToCreatePassword, err.Error())
```

Comment:
We can't use fatal. Please think about the response to the frontend. You should add return etc. here.

Reply:
Actually, I thought to change Password() function return parameters, do not you think to include `error` parameter to Password() function ?

Reply:
I think, it can be usefully when debugging comes to place.

Reply:
```
	func GeneratePassword() (string, error) {
              length := viper.GetInt("server.generatedPasswordLength")
               res, err := password.Generate(length, 10, 10, false, false) 
               if err != nil {
                      return "", err 
                }
               return res, nil 
        }
```
Of course, all calls will be updated when this is changed ! 

@yakuter

Reply:
I have updated it, I hope, you find it suitable.